code/go/0chain.net/blobbercore/challenge/map.go

```diff
@@ -0,0 +1,45 @@
+package challenge
+
+import (
+	"sync"
+)
+
+var cMap = &ChallengeMap{
+	items: make(map[string]ChallengeStatus),
```

Comment:
Use the LRU cache to avoid potential OOM, https://github.com/0chain/blobber/blob/6041fff4855a9ffee008b4bf8ec5e6ff5b7b3caa/code/go/0chain.net/core/cache/lru.go#L13.

We should be able to use it directly to replace the ChallengeMap

Reply:
fixed